release_notes.md

```diff
@@ -1,5 +1,6 @@
 ## next (unreleased)
 
+- [#781](https://github.com/Flank/flank/pull/781) Create results-dir. ([adamfilipow92](https://github.com/adamfilipow92)) 
```

Comment:
The bug is, we should not call `assertFileExists` on `gcloud.resultsDir`. I'd update the release notes to say something like:

>  [#781](https://github.com/Flank/flank/pull/781) Remove local exists check on cloud results-dir. Fixes crash when resultsDir is set by the user. ([adamfilipow92](https://github.com/adamfilipow92))

Reply:
Changed!